src/Raven.Server/Documents/Handlers/ReplicationHandler.cs

```diff
@@ -236,10 +235,64 @@ public Task Performance()
             return true;
         }
 
+        [RavenAction("/databases/*/replication/pulses/live", "GET", AuthorizationStatus.ValidUser, SkipUsagesCount = true)]
+        public async Task PulsesLive()
```

Comment:
Based on types:
https://github.com/ravendb/ravendb/pull/3151/files#diff-7ba6900cefee273c07cc76b10b9c7265R93

I think naming it hearbeat is not correct, as we need some broader term. Maybe: 'event' , or taskEvent?